interface-definitions/system-display.xml.in

```diff
@@ -0,0 +1,245 @@
+    system display show host (cpu|cpu-all|cpu-hist|disk|load-hist|memory|proc|uptime)
+                        network interface <intName> alias <alias>
+                                units (bps|Bps|pps)
+                        clock (big|mini|date-time)
+                        title <name>
+
+    system display duration <s>
+    system display hello <string>
+    system display bye <string>
+    -->
+  <node name="system">
+    <children>
+      <node name="display" owner="${vyos_conf_scripts_dir}/system-display.py">
+        <properties>
+          <help>System display LCD/VFD/LED</help>
+          <priority>400</priority>
+        </properties>
+        <children>
+          <!-- system display model (SDEC|EZIO) -->
+          <leafNode name="model">
+            <properties>
+              <help>Model of the display attached to this system [REQUIRED]</help>
+              <completionHelp>
+                <list>SDEC EZIO</list>
+              </completionHelp>
+              <constraint>
+                <regex>(SDEC|EZIO)</regex>
+              </constraint>
+              <constraintErrorMessage>Invalid system display model</constraintErrorMessage>
+              <valueHelp>
+                <format>SDEC</format>
+                <description>Display model for Lanner, Watchguard, Nexcom NSA, Sophos UTM appliances with built-in SDEC LCD</description>
+              </valueHelp>
+              <valueHelp>
+                <format>EZIO</format>
+                <description>Display model for Portwell, Caswell appliances with built-in EZIO-100 or EZIO-300 LCD</description>
+              </valueHelp>
+            </properties>
+          </leafNode><!--system display model-->
+          <!-- system display config (enabled|disabled) -->
+          <leafNode name="config">
+            <properties>
+              <help>Configuration status</help>
+              <completionHelp>
+                <list>enabled disabled</list>
+              </completionHelp>
+              <constraint>
+                <regex>(enabled|disabled)</regex>
+              </constraint>
+              <constraintErrorMessage>Invalid system display config status</constraintErrorMessage>
+              <valueHelp>
+                <format>enabled</format>
+                <description>Turn system display configuration ON</description>
+              </valueHelp>
+              <valueHelp>
+                <format>disabled</format>
+                <description>Turn system display configuration OFF</description>
+              </valueHelp>
+            </properties>
+          </leafNode><!--system display config-->
+          <node name="show">
+            <properties>
+              <help>Select the screens for the system display [REQUIRED]</help>
+            </properties>
+            <children>
+              <!-- system display show host (cpu|cpu-all|cpu-hist|disk|load-hist|memory|proc|uptime) -->
+              <leafNode name="host">
```

Comment:
I'm not yet confident if we really should expose "everything" that is configurable to the user - this will just add additional complexity. Why don't we decide which should be displayed?

Reply:
Also I'm missing the VyOS version information here - this is sometimes very useful if there is a remote guy and tells you "device is malfunctioning" but he can't tell us the version.

Reply:
I suppose we can decide on these options a bit later when you get to try them on your own LCD.

For the version, are you asking to show the VyOS version on the LCD somehow?